NeverGoStable/wellness
# Login (web)

#### What's this do?

* Adds initial redux stuffs
* Adds token reducer
* Tokens are stored and bootstrapped via `localStorage`

#### Why are we doing this?

This is part of #13 

#### Who should review this PR?

@joeyliu2012 @ak493812 @tm507211 @wilsonmct55 

#### Anything the reviewer(s) should know to review effectively?

This currently doesn't address routing or using the auth tokens for subsequent requests. I will work on that in another PR.

#### How should this be tested?

`npm install && npm run web`

Then open up localhost:3000. Create an account and then attempt to login to that account. You should have a `X-AUTH-TOKEN` in local storage and see some logging from redux.

#### Dependencies for merging? Releasing to production?

None.

#### Screenshots (if applicable)

![image](https://cloud.githubusercontent.com/assets/1699281/10143969/2550e190-65e9-11e5-94a1-f5846436474f.png)

![image](https://cloud.githubusercontent.com/assets/1699281/10143990/2e5bdb50-65e9-11e5-93a5-abd2e3fe0f88.png)

![image](https://cloud.githubusercontent.com/assets/1699281/10144006/42c2425a-65e9-11e5-94e0-f6aa6982ce38.png)
